app/policies/facility_policy.rb

```diff
@@ -48,7 +77,22 @@ def initialize(user, scope)
     end
 
     def resolve
-      scope.where(facility_group: user.facility_groups)
+      if user.has_permission?(:can_manage_all_organizations)
+        return scope.all
+      elsif user.has_permission?(:can_manage_an_organization)
+        facility_groups = resources_for_permission(:can_manage_an_organization).flat_map(&:facility_groups)
```

Comment:
Perhaps this could also follow the same style as the other else branches? I mean like so: `return scope.where(facility_groups: ...)`